notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        "gs://automl-tables-build-oss-dependencies/dataset/safe_driver/train.csv"
</pre><p>Training datasets must be in the right location. See https://docs.google.com/document/d/12Nwws5YpbU5D5zC7u08Bqpz6y8DBVt4lm21IQswUruc/edit#bookmark=id.iip5an12bybl</p><p> </p><p>Let me know where you want to copy it and I can do it for you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Thanks Ivan, could you help put it to gs://cloud-samples-data/vertex-ai/tabular-workflows/datasets/safe-driver/train.csv?